docs/userGuide/reusingContents.md

```diff
@@ -59,14 +59,12 @@ If you are using Git for version control, you can set up the sub-site repository
 </span>
 </include>
 
+#### Excluding Files from Rendering as Pages
 
-<hr><!-- ======================================================================================================= -->
-
-## Creating Custom Fragments
-
-**MarkBind supports creating custom fragments**. A fragment is a piece of content that can be reused across multiple pages. This allows you to create reusable content similar to [reusing content](#reusing-contents-across-sites), but rather than reusing content from a _sub-site_, content is reused from fragments which can be excluded from page generation.
+**MarkBind supports the exclusion of files from rendering as pages**. This allows for the ease of creation of <tooltip content="A fragment is a piece of content that can be reused across multiple pages.">custom fragments</tooltip>. This allows you to create reusable content similar to reusing content, but rather than reusing content from a _sub-site_, content is reused from fragments which are excluded from page generation. This facilitates the use of repeated content across the site.
```

Comment:
@itsyme Could we perhaps rephrase this further to make it clearer? It seems a little lengthy and the link to "reusing content" has been removed, which makes this part confusing:
> This allows you to create reusable content similar to reusing content, but rather than reusing content from a _sub-site_, content is reused from fragments which are excluded from page generation.

wdyt?

Reply:
I agree about the lengthiness! I will rephrase it to keep it more succinct. As for the removal of the link, I initially agreed with what @lhw-1 said above, the reasoning being that this section is already under the reusing content section. However, I would be open to inserting the link back as well to make it seem clearer.